### PR TITLE
Suggest wiping your environment to folks with bad build outcomes.

### PR DESCRIPTION
**Pull request by ericholscher**
This is a basic start to trying to be more helpful to users.

### CODE REVIEW

**Comment by humitos** [comment]
I like the idea of giving some tips/advice on the "why this build has failed?". I would like to start testing something simple like this, how it goes, and maybe later try to make it "more intelligent" but I wouldn't spend more time on the intelligence at the moment but maybe more tips.

**Comment by ericholscher** [comment]
Aye. I think we could build some nicer styles, and have easier ways for users to contribute them (eg. a YAML file of possible errors). But this is a nice start.

File: readthedocs/templates/wipe_version.html [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+  {% endwith %}
+{% endblock %}
+
+
 {% block title %}Wipe the build directories for a version {% endblock %}
 
 {% block content %}
 
+{% blocktrans with slug=version.slug %}
+    <h3>Remove build environment for <em>{{ slug }}</em> version.</h3>
```
Comment (agjohnson) [inline readthedocs/templates/wipe_version.html]: I'd remove the h3 at least from the blocktrans

File: readthedocs/templates/wipe_version.html [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+    {% endblocktrans %}
 {% else %}
-Having trouble getting your documentation build to complete?  By clicking on the button below you'll clean out your build checkouts and environment, but not your generated documentation.
+    {% blocktrans %}
+        Having trouble getting your documentation build to complete?  By clicking on the button below you'll clean out your build checkouts and environment, but not your generated documentation.
+    {% endblocktrans %}
 
 <form method="post" action="#">
-  <input type="submit" value="Wipe version">
+  <input type="submit" value="Wipe {{ version.slug }}">
```
Comment (agjohnson) [inline readthedocs/templates/wipe_version.html]: Non-translated copy here

File: readthedocs/templates/builds/build_detail.html [hunk 3]
```diff
@@ -92,6 +92,29 @@
...
+        {% endblocktrans %}
+      </p>
+    {% endif %}
+
+    {% if not build.success and "setup.py install" in build.commands.last.output %}
+      <p>
+        {% url 'projects_advanced' build.version.project.slug as advanced_url %}
+        {% blocktrans %}
+        Don't want <pre>setup.py install</pre> called?
+        Change the <strong>Install Project</strong> setting in your <a href="{{ advanced_url }}">advanced settings</a>.
```
Comment (agjohnson) [inline readthedocs/templates/builds/build_detail.html]: `advanced_url` should be in the `blocktrans` variable list

File: readthedocs/templates/builds/build_detail.html [hunk 4]
```diff
@@ -92,6 +92,29 @@
...
     </div>
 
+    <div class="build-ideas">
+    {% if not build.success and build.commands.count < 4 %}
+      <p>
+
+        {% url 'wipe_version' build.version.project.slug build.version.slug as wipe_url %}
+        {% blocktrans %}
+        Having trouble with your build environment?
+        Try <a href="{{ wipe_url }}">reseting it</a>.
```
Comment (agjohnson) [inline readthedocs/templates/builds/build_detail.html]: `wipe_url` should be in the blocktrans variable list

File: readthedocs/templates/wipe_version.html [hunk 5]
```diff
@@ -1,16 +1,31 @@
...
 {% block content %}
 
+{% blocktrans with slug=version.slug %}
+    <h3>Remove build environment for <em>{{ slug }}</em> version.</h3>
+{% endblocktrans %}
+
 {% if deleted %}
-Your project environment has been wiped.<br><br>
+    {% blocktrans %}
+        Your project environment has been wiped.<br><br>
```
Comment (agjohnson) [inline readthedocs/templates/wipe_version.html]: All `<br>` can be removed. Spacing should be forced with a surrounding `p` element, or other css.

File: readthedocs/templates/builds/build_detail.html (hunk 4 above)
Comment (ericholscher) [reply readthedocs/templates/builds/build_detail.html]: This is how the docs say to do it. https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#blocktrans-template-tag

**Pull request merged**